sdks/java/core/src/main/java/com/google/cloud/dataflow/sdk/io/BigQueryIO.java

```diff
@@ -24,6 +24,7 @@
 import com.google.api.services.bigquery.model.TableSchema;
```

Comment:
can you add unit tests for the new code in this file? for the writer, write operation, etc?